gcp-cloud-run/01-variables.tf

```diff
@@ -167,119 +152,153 @@ variable "pvault_log_customer_env" {
   type        = string
 }
 
-variable "vault_sql_serverless_connector_range" {
+variable "pvault_sql_serverless_connector_range" {
   description = "Cloud Run connector /28 CIDR range (used to connect Cloud Run to VPC)"
   type        = string
   default     = "10.8.0.0/28"
 }
 
+variable "pvault_devmode" {
+  description = "Enable devmode for Pvault. See https://piiano.com/docs/guides/configure/environment-variables#production-and-development-mode for more details."
+  type        = bool
+  default     = false
+}
+
+variable "pvault_env_vars" {
+  description = "A map of PVAULT environment variables"
+  type        = map(string)
+  default     = {
+    # Add environment variables as needed, for example:
+    # PVAULT_FEATURES_MASK_LICENSE = true
+  }
+}
+
 variable "connector_cloud_run_max_instances" {
   description = "Maximum number of instances used by VPC Serverless connector"
   type        = number
   default     = 4
 }
 
-########################
-### Vault CLI Server ###
-########################
-variable "vault_cli_subnet" {
-  description = "Subnet where Vault CLI will be deployed"
+###########################
+### Cloud Load Balancer ###
+###########################
```

Comment:
Will those resources be created if the proxy is not created? If not, better clarify the description